pages/downloads.md

```diff
@@ -86,15 +86,14 @@ available in a future update to this site.
 {% assign lnkFile = 'accumulo-' | append: linkVers | append: '-' | append: srcbin | append: '.tar.gz' %}
 {% assign lnkSuffix = '/accumulo/' | append: linkVers | append: '/' | append: lnkFile %}
 <div class="row btn-group" style="margin-left: 20px; margin-bottom: 5px; display: block">
-  <div class="col btn-group">
+  <div class="col-auto btn-group">
     <a {{btnDownloadStyle}} href="{{closerLink}}{{lnkSuffix}}" link-suffix="{{lnkSuffix}}">{{lnkFile}}{{glyphSave}}</a>
-  </div><div class="col btn-group">
```

Comment:
You removed the separate div, but if I remember correctly, this was here to ensure the entire block wrapped as a group on screen width changes. Or maybe it was because it was for padding/spacing tweaks. I don't think it was extraneous, though. Same situation in the others below here.

Reply:
Fixed in b056d84